client/src/main/java/com/aws/greengrass/cli/commands/DeploymentCommand.java

```diff
@@ -131,23 +134,75 @@ public DeploymentCommand(NucleusAdapterIpc nucleusAdapterIpc) {
     }
 
     // GG_NEEDS_REVIEW: TODO: input validation and better error handling https://sim.amazon.com/issues/P39478724
+    /*
+    Output example:
+    Deployment ID: Deployment UUID
+    Created on: Time in UTC when the deployment was created (dd-MM-uuuu HH:mm:ss UTC)
+    Status: DEPLOYMENT_STATUS
+    Deployment Status Details: DETAILED_DEPLOYMENT_STATUS
```

Comment:
I'm still a little bothered by the duplication of status and detailed status - would it make sense to have something like
```
status = DETAILED_DEPLOYMENT_STATUS != null ? DETAILED_DEPLOYMENT_STATUS : DEPLOYMENT_STATUS
```

Reply:
Yeah, I dont see the point of having tow successful/failed statuses at the same time, can replace this. But have to make sure that `IN_PROGRESS` remains in the output.